### DISCUSSION TITLE
Enhance the mobile UI

### DISCUSSION THREAD

**Discussion by youbian89** [TriliumNext/Trilium#3270] [Q&A]
Hi, I like Trilium very much. Since there is no plan to mak Android app, I modify some source code for my purpose to use Trilium more conveniently. When I first used the Trilium on Android phone Browser in mobile version,I meet with Bad experience :
1.**add background-color for expand icon**（"<" icon）
When I just want to open the node to see the child nodes, I click the icon of the node on its left, but... it is usally open the node detail while not the child nodes.  So, I modify the moblile layout to **add  background-color for expand icon**（"<" icon）. More accurate positioning~

2.**Adding two buttons to switch between node tree and note detail**.
It is very useful when I just want to hava a look at other node nameand come back to current note quickly.(for some purposer, eg: maybe , current note need the title of other note name to show something or mark some relations ) . 
in fact, if the treewidget can hide and display freely as a free html layer ande control by a button,it will be perfect. (something like bootstrap dropdown menu). I am not a Professional Programmer, so I can not realize it. ^^

Picture reference ：
tag 1: add  background-color for expand icon
tag 2:add switch button (left :switch to content tree    right :switch to note detail )
![20221030164041](https://user-images.githubusercontent.com/48555962/198869936-87c325ee-e583-480a-bd06-917ed039e024.jpg)

I have uploaded the src files. (moile.js is modified ，other files are new files) 
structure： 
```
src
|__app
    |__mobile.js   "new MobileLayout()" =>  "new MobileLayout_v2()"
    |__layouts
        |__mobile_layout_v2.js (new file)
    |__widgets
        |__quick_search_v2.js (new file)
        |__mobile_widgets
            |__disp_content_button.js (new file)
            |__disp_detail_button.js (new file)
            |__mobile_global_buttons_v2.js (new file)
```

If you test ok and feel good，I wish you could add the modify to Later versions. 
 If so ,I need not modify the source code when new versions released.   haha^^

Thank you!

[src.zip](https://github.com/zadam/trilium/files/9895447/src.zip)

**Comment by youbian89** [comment]
@zadam Hi,will you consider to replace the source files ,to use the new mobile UI ? Thank you

**Reply by zadam** [reply]
Sorry, I don't plan to incorporate these changes into the master version.
